dashboard/routes/batch.py

```
@@ -123,27 +124,125 @@ def get(batch_name: str, chart_index: int):
+
+    Returns:
+        Div: The expanded chart.
+    """
+    df = app.state.df_cache[batch_name]
+    metric_stats = app.state.stats_cache[batch_name]
+    metric_name = metric_stats[chart_index]["metric_name"]
+
+    # Generate expanded chart (larger and with more interactive features)
+    df_metric = df[df["metric_name"] == metric_name]
+    df_metric = extract_metadata(df_metric, "anomaly_explanation")
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)